<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Préférences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/icon</normaloff>:/icones/icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sortie audio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboAudioOuput"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gestion RAM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboRam">
       <item>
        <property name="text">
         <string>Charger en mémoire lorsque nécessaire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tout charger en mémoire</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoucle">
       <property name="text">
        <string>ajuster à la boucle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBlanc">
       <property name="text">
        <string>enlever le blanc au départ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Import fichiers wav</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushFermer</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboAudioOuput</sender>
   <signal>activated(int)</signal>
   <receiver>Config</receiver>
   <slot>setAudioOutput(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboRam</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Config</receiver>
   <slot>setRam(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoucle</sender>
   <signal>toggled(bool)</signal>
   <receiver>Config</receiver>
   <slot>setWavAutoLoop(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBlanc</sender>
   <signal>toggled(bool)</signal>
   <receiver>Config</receiver>
   <slot>setWavRemoveBlank(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRam(int)</slot>
  <slot>setAudioOutput(int)</slot>
  <slot>setWavAutoLoop(bool)</slot>
  <slot>setWavRemoveBlank(bool)</slot>
 </slots>
</ui>
